Directory.Packages.props

```diff
@@ -90,7 +90,7 @@
 
 		<PackageVersion Include="Microsoft.NET.Test.Sdk" Version="17.12.0" />
 		<PackageVersion Include="Microsoft.NETCore.Jit" Version="2.0.8" />
-		<PackageVersion Include="Microsoft.Playwright.Xunit" Version="1.50.0-beta-2" />
+		<PackageVersion Include="Microsoft.Playwright.Xunit" Version="1.50.0" />
```

Comment:
Why is this part of this PR?

Reply:
https://github.com/DuendeSoftware/products/pull/1875 will rebase after this is merged.